source/P0323R9.bs

```diff
@@ -987,8 +992,8 @@ otherwise
 
 *Throws*: Any exception thrown by the operations specified in the effect clause.
 
-*Remarks*: If an exception is thrown during the call to `T`'s assignment nothing
-changes.
+*Remarks*: If an exception is thrown during the call to `T`'s assignment, `*this` remains
+unchanged.
```

Comment:
L-value arguments passed by r-value may be in a post-moved state (thus changed) after being emplaced into the `tmp` objects.